tendrl/node_agent/tests/test_alerts_utils.py

```diff
@@ -0,0 +1,23 @@
+from tendrl.node_agent.manager.alert_socket import AlertUtils
+import json
+import pytest
+
+
+class Test_alerts_utils(object):
+    def test_alertsSuccessfulValidate(self):
+        expected_alert = {
+            'Host': '127.0.0.1',
+            'Plugin': 'cpu',
```

Comment:
Is this "Plugin" value expected of al alerts that collectd writes to the node agent socket?

Also, ceph_integration and gluster_integration will be writing alerts to the node agent socket, 

You willl need to have a generic alert json schema using which any process can write alerts to the node agent socket